frontend/components/Dashboard/Nav/Nav.tsx

```diff
@@ -48,7 +47,7 @@ const Nav: React.FC<Props> = ({ currentUser, expanded, collapse }) => {
           width: 100%;
           background-color: rgba(0, 0, 0, 0.4);
           opacity: 0;
-          transition: opacity ${navConstants.transitionDuration}ms ease-in-out;
+          transition: opacity ${navConstants.transitionDuration}ms linear;
```

Comment:
Other animations were using `linear`, so making them all the same easing function helps the animations look more in sync. We can play play around with the exact easing function and update all of the locations at once, if need be